lib/llvm-backend/src/platform/common.rs

```diff
@@ -0,0 +1,3 @@
+pub fn round_up_to_page_size(size: usize) -> usize {
```

Comment:
Page size is not always 4kib on any given system. Windows and Linux both support different sized pages in the same running system.

On unix, it's `libc::sysconf(_SC_PAGESIZE)` but not on Windows, which suggests maybe this isn't really platform common.

Reply:
This was same code as before, just moved into a new file.
Perhaps we can address this in a separate PR?